data-esnext.js

```diff
@@ -92,6 +92,12 @@ exports.tests = [
     ie11: false,
     firefox10: false,
     firefox52: false,
+    firefox103: false,
+    firefox104: {
+          val: 'flagged',
+          note_id: 'ff-shadow-realm',
+          note_html: 'The feature has to be enabled via <code>javascript.options.experimental.shadow_realms</code> setting under <code>about:config</code>.'
+        },
```

Comment:
Hmm... Probably spec has changed since then. Maybe we need to reflect that in the comment

Reply:
The last commit for this feature was on Firefox 107, I updated the patch to this version.